WordPress/Classes/ViewRelated/Reader/Sidebar/ReaderSidebarViewModel.swift

```diff
@@ -14,14 +14,26 @@ final class ReaderSidebarViewModel: ObservableObject {
     private var isRestoringSelection = false
 
     @Published var isCompact = false
+    let isReaderModeEnabled: Bool
```

Comment:
Does "Reader Mode" mean in the Reader app? If that's the case, should we name this property `isInReaderApp` or something to that effect?

Reply:
Sounds good. Let's rename it to `isReaderAppModeEnabled`. I'm planning to remove these as soon as we can.